### PR TITLE
HIVEMALL-14: Build instruction including xgboost library

### PR DESCRIPTION
**Pull request by Lewuathe**
see: https://issues.apache.org/jira/browse/HIVEMALL-14

### CODE REVIEW

File: src/site/markdown/contributing.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+#### Prerequisites
+
+* Maven 3.x
+* JDK 1.7 or later
+
+```
+$ cd hivemall
+
+# This is a workaround for resolving xgboost dependencies.
+$ mvn validate -Pxgboost
```
Comment (maropu) [inline src/site/markdown/contributing.md]: We need `-Pxgboost`?

**Comment by maropu** [comment]
I left a minor comment though, LGTM

File: src/site/markdown/contributing.md (hunk 1 above)
Comment (Lewuathe) [inline src/site/markdown/contributing.md]: Without `-Pxgboost`, we should face dependency error. And this profile is also set in [travis environment](https://github.com/myui/hivemall/blob/master/.travis.yml#L31).

```
Could not resolve dependencies for project io.github.myui:hivemall-xgboost:jar:0.4.2-rc.2: Could not find artifact ml.dmlc:xgboost4j:jar:0.10 in cloudera (https://repository.cloudera.com/artifactory/cloudera-repos/
```

**Comment by myui** [comment]
Thanks. Merged.

**Pull request merged**